### ISSUE TITLE
Legacy flow in DHF5 final to final not working

### ISSUE THREAD

**Issue by ltnd** [marklogic/marklogic-data-hub#2949]
The dhfwriter is not working when doing legacyFlow from final to final.

What we saw : 
=> when doing Staging to final : the module « /data-hub/4/extensions/sjsflow.sjs » in the MODULE db is used and it's working.
=> when doing Final to final : the extension included in MarkLogic Server dans MarkLogic is used  (for unix, file /opt/MarkLogic/Modules/MarkLogic/data-hub-framework/extensions/sjsflow.sjs).

Updating the file /opt/MarkLogic/Modules/MarkLogic/data-hub-framework/extensions/sjsflow.sjs with the following lines (using the same require as /data-hub/4/extensions/sjsflow.sjs) :
```
//const config = require("/com.marklogic.hub/config.sjs");
//const consts = require("/MarkLogic/data-hub-framework/impl/consts.sjs");
//const flowlib = require("/MarkLogic/data-hub-framework/impl/flow-lib.sjs");
//const tracelib = require("/MarkLogic/data-hub-framework/impl/trace-lib.sjs");
 
const config = require("/com.marklogic.hub/config.sjs");
const consts = require("/data-hub/4/impl/consts.sjs");
const flowlib = require("/data-hub/4/impl/flow-lib.sjs");
const tracelib = require("/data-hub/4/impl/trace-lib.sjs");
```
we succeed to write in FINAL from FINAL.

**Comment by ryanjdew** [comment]
Thank you for filing an issue. This is related to internal ticket https://project.marklogic.com/jira/browse/DHFPROD-3068

**Comment by kkanthet** [comment]
Issue Fixed in 5.0.3 release
